app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::V1::CommentsController < Api::V1::ApplicationController
+  def index
+    @post = Post.find(params[:post_id])
+
+    render json: @post.comments
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
+
+    if @comment.save
+      render json: @comment, status: :created
+    else
+      render json: @comment.errors, status: :unprocessable_entity
+    end
+  end
```

Comment:
- [ ] As mentioned in the requirements : 

![image](https://user-images.githubusercontent.com/89282221/218089413-b670788c-9cf0-449e-91a2-513c34373e18.png)

you should user current user instead 🙏

Reply:
Its not clear what U mean @ichala can you give more hints?

Reply:
We Talked on slack 💯